customizations/alma_disable_cloudinit.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+# TEMP add to fix changed behavior in AlmaLinux 8.7
```

Comment:
can we add that the behavior is that cloud-init keeps reverting the /etc/sysconfig/network-scripts/ifcfg-eth0